install_config/certificate_customization.adoc

```diff
@@ -29,72 +29,300 @@ custom certificates can be configured using the
+
+To configure this, add the xref:../install_config/install/advanced_install.adoc#advanced-install-configuring-docker-route[following lines] 
+to the Ansible inventory file, by default *_/etc/ansible/hosts_* file, in the `[OSEv3:vars]`
+section. pointing to the certificates you would like used for the registry route.
+
+----
+openshift_hosted_registry_routehost=registry.apps.c1-ocp.myorg.com <1>
+openshift_hosted_registry_routecertificates={"certfile": "/path/to/registry.apps.c1-ocp.myorg.com.crt", "keyfile": "/path/to/registry.apps.c1-ocp.myorg.com.key", "cafile": "/path/to/registry.apps.c1-ocp.myorg.com-ca.crt"} <2>
+openshift_hosted_registry_routetermination=reencrypt <3>
+----
+
+//https://github.com/openshift/openshift-docs/issues/5765
+<1> The host name of the registry
+<2> The locations of the *cacert*, *root*, and *cafile* files.
++
+* *certfile* contains the {product-title} custom router certificate.
+
+* *keyfile* contains the {product-title} custom key
+
+* *cafile* contains the root CA for this key and certificate. If an intermediate CA is in use it should contain both the intermediate and root CA.
+<3> Specify where encryption is performed:
++
+* Set to `reencrypt` with a _re-encrypt route_ to terminate encryption at the edge router and re-encrypt it with a new certificate supplied by the destination.
++
+* Set to `passthrough` to terminate encryption at the destination. The destination is responsible for decrypting traffic.
+
+[[configuring-custom-certificates-retrofit]]
+== Retrofit Custom Certificates into a Cluster
+
+You can retrofit custom master and custom router ceritificates into an existing {product-title} cluster by editing the 
+the Ansible inventory file, by default *_/etc/ansible/hosts_* and running a playbook.
+
+[[configuring-custom-certificates-retrofit-master]]
+=== Retrofit Custom Master Certificates into a Cluster
+
+To retrofit custom certificates: 
+
+. Edit the Ansible inventory file to set the `openshift_master_overwrite_named_certificates=true` and specify the path to the certificate using `openshift_master_named_certificates`. 
++
+[source,yaml]
+----
+openshift_master_overwrite_named_certificates=true
+openshift_master_named_certificates=[{"certfile": "/path/on/host/to/crt-file", "keyfile": "/path/on/host/to/key-file", "names": ["public-master-host.com"], "cafile": "/path/on/host/to/ca-file"}] <1>
+----
++
+<1> Path to a xref:configuring-custom-certificates-master[master API certificate]. 
+
+. Run the following playbook:
++
+----
+ansible-playbook /usr/share/ansible/openshift-ansible/playbooks/redeploy-certificates.yml
+----
+
+[[configuring-custom-certificates-retrofit-router]]
+=== Retrofit Custom Router Certificates into a Cluster
+
+To retrofit custom router certificates: 
+
+. Edit the Ansible inventory file to set the `openshift_master_overwrite_named_certificates=true` and specify the path to the certificate using `openshift_hosted_router_certificate`.
```

Comment:
s/ using `openshift_hosted_router_certificate`./ as the value of the `openshift_hosted_router_certificate` parameter.